### PR TITLE
Adding tests for new docker build-based OpenShift deployment strategy

### PR DESCRIPTION
**Pull request by mjurc**
…, both for serverless services and classic deployments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,6 +312,14 @@ In such case, the `reason` attribute of the annotation should point to correspon
 If the `@OpenShiftTest` annotation is configured to use the strategy `UsingQuarkusPluginDeploymentStrategy` and the module contains the `quarkus-openshift` Quarkus extension, the test framework will use Quarkus to build and deploy the module into OpenShift.
 Note that the property `quarkus.kubernetes.deploy` must NOT be set in the module configuration in order to avoid to perform the deployment at build time.  
 
+#### Deployment using Quarkus docker build strategy
+
+If the `@OpenShiftTest` annotation is configured to use the strategy `UsingQuarkusPluginDeploymentStrategy.UsingQuarkusPluginDeploymentStrategy`
```
Comment (llowinge) [inline README.md:317]: @mjurc Probably typo in `UsingQuarkusPluginDeploymentStrategy. UsingDockerStrategy`

**Comment by rsvoboda** [comment]
run tests

**Comment by llowinge** [comment]
Hi @mjurc @rsvoboda any plans to merge it ?

**Comment by rsvoboda** [comment]
OCP clusters are still blocked due to PSI issues :/ Quarkus main is not build-able due to unavailable hibernate reactive artifact.

So we are blocked before the final spin.

**Comment by rsvoboda** [comment]
please rebase

**Comment by mjurc** [comment]
run tests

**Comment by mjurc** [comment]
@rsvoboda This has been rebased, and tests work now:

https://main-jenkins-csb-quarkusqe.apps.ocp4.prod.psi.redhat.com/job/openshift-test-suite-ci/488/
https://main-jenkins-csb-quarkusqe.apps.ocp4.prod.psi.redhat.com/job/openshift-test-suite-native-ci/343/

Can this be merged now?

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**